tap_nice_incontact/streams.py

```diff
@@ -803,6 +803,24 @@ class QMWorkflows(DataExtractionStream):
     default_period = 'days'
 
 
+class QMQuestions(DataExtractionStream):
+    """
+    Retrieve QM Questions via the data extraction api.
+
+    Docs: https://help.nice-incontact.com/content/recording/dataextractionapi.htm#QMQuestionsandAnswersQAEntityandCSVFile
+    """
+    tap_stream_id = 'qm_questions'
+
+    # Entity information required to create a job
+    entity_name = 'qm-questions'
+    entity_version = 1
+
+    key_properties = ['evaluationSubmissionDate', 'workflowinstanceId']
```

Comment:
Wouldn't `questionId` make more sense here? `evaluationSubmissionDate` and `workflowInstanceId` says they can both be null which can result in issues.

Reply:
yes good catch, updating the `key_properties` to be `['workflowinstanceId', 'questionId']` and making both of those fields non-nullable